azure/services/aso/aso.go

```diff
@@ -120,6 +135,21 @@ func (s *Service) CreateOrUpdateResource(ctx context.Context, spec azure.ASOReso
 	parameters.SetName(resourceName)
 	parameters.SetNamespace(resourceNamespace)
 
+	labels := make(map[string]string)
+	anns := make(map[string]string)
```

Comment:
not a fan of the abbreviation, annotations is more clear/explicit to me (but feel free to disagree)